lib/src/widgets/adw/header_bar.dart

```diff
@@ -215,15 +202,11 @@ class _AdwHeaderBarState extends State<AdwHeaderBar> {
                       leading: Row(
                         mainAxisSize: MainAxisSize.min,
                         children: [
-                          if (hasWindowControls &&
-                              sep != null &&
-                              sep[0].split(',').isNotEmpty) ...[
+                          if (hasWindowControls && sep != null && sep[0].split(',').isNotEmpty) ...[
                             SizedBox(width: widget.style.titlebarSpace),
                             for (var i in sep[0].split(','))
                               if (windowButtons[i] != null) windowButtons[i]!,
-                            if (!widget.style.nativeControls ||
-                                !kIsWeb && Platform.isLinux)
-                              SizedBox(width: widget.style.titlebarSpace),
+                            if (!widget.style.nativeControls || !kIsWeb && Platform.isLinux) SizedBox(width: widget.style.titlebarSpace),
```

Comment:
Can you please format this file with line characters as 80, see this for more:
https://dart.dev/tools/linter-rules/lines_longer_than_80_chars